src/components/LoadingOverlay/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { CircularProgress, Backdrop } from '@mui/material';
+
+interface LoadingOverlayProps {
+  isLoading: boolean;
+}
+
+const Constants = {
+  circularProgressSize: 100,
+};
```

Comment:
**thought:** if we're not going to have more constants right now, maybe we should keep as a single variable instead of an object.